backend/src/plugins/Mutes/functions/muteUser.ts

```diff
@@ -91,6 +91,7 @@ export async function muteUser(
       rolesToRestore = currentUserRoles.filter((x) => (<string[]>restoreRoles).includes(x));
     }
 
+    const zep = await pluginData.guild.members.fetchMe();
```

Comment:
Why this move? It's not used outside of the scope it was on previously

Reply:
yes